From df7b90ef98794ca956f4be245157551244fe2bd5 Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Thu, 19 Mar 2015 11:15:57 +0200
Subject: [PATCH 0158/1240] doimage: updated uart configuration in header

- No need to set the uart mode to update on baudrate update
- This enables indpendently updating the buadrate and port

Change-Id: Id3ccde0224eae46f55fc171fdfd7c6c73eb41da8
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/17656
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 tools/doimage.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/tools/doimage.c b/tools/doimage.c
index 6aa3f12..a6d3354 100644
--- a/tools/doimage.c
+++ b/tools/doimage.c
@@ -551,15 +551,11 @@ void update_uart(header_t *header)
 	header->uart_cfg = 0;
 	header->baudrate = 0;
 
-	if (opts.disable_print) {
+	if (opts.disable_print)
 		uart_set_mode(header->uart_cfg, UART_MODE_DISABLE);
-		printf(" Disabling UART 0x%x\n", header->uart_cfg);
-	}
 
-	if (opts.baudrate) {
-		uart_set_mode(header->uart_cfg, UART_MODE_UPDATE);
+	if (opts.baudrate)
 		header->baudrate = (opts.baudrate / 1200);
-	}
 }
 
 /* ****************************************
-- 
1.9.1

